### PR TITLE
Remove profile_image_url from the person model, it is no longer neeed

### PR DESCRIPTION
**Pull request by staubina**
[DISCO-283]

As part of the work to make course_metadata the source of truth,
we have moved all profile_images to the Discovery data store and
are now using the profile_image field exclusively to store the
URL and information.

### CODE REVIEW

File: course_discovery/apps/api/serializers.py [hunk 1]
```diff
@@ -290,6 +290,9 @@ def get_social_network_url(self, url_type, obj):
         if social_networks:
             return social_networks[0].url
 
+    def get_profile_image_url(self, obj):
+        return obj.get_profile_image_url
+
```
Comment (mikix) [inline course_discovery/apps/api/serializers.py:295]: Why this change? Same result right?

**Review by mikix — APPROVED** [review]

File: course_discovery/apps/api/serializers.py (hunk 1 above)
Comment (staubina) [reply course_discovery/apps/api/serializers.py:295]: It was having issues returning the correct url with the old serializer. This update also matches the approach in another serializer below this one.
